qa-tester/src/Admin_Bar.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Class Admin_Bar.
+ *
+ * @package AmpProject\AmpWP_QA_Tester
+ */
+
+namespace AmpProject\AmpWP_QA_Tester;
+
+/**
+ * Class handling the plugin's admin bar menu.
+ *
+ * @since 1.0.0
+ */
+class Admin_Bar {
+
+	/**
+	 * Build ID of the currently installed plugin version.
+	 *
+	 * @since 1.0.0
+	 * @var string
+	 */
+	protected $build_id;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 1.0.0
+	 */
+	public function __construct() {
+		$this->build_id = get_site_option( Plugin::ID_STORAGE_KEY );
+	}
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @since 1.0.0
+	 */
+	public function register() {
+		add_action( 'admin_bar_menu', [ $this, 'add_menu_button' ], 99 );
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_plugin_assets' ] );
+	}
+
+	/**
+	 * Enqueue the plugin assets.
+	 *
+	 * @since 1.0.0
+	 */
+	public function enqueue_plugin_assets() {
+		// Only active if the admin bar is showing.
+		if ( ! is_admin_bar_showing() ) {
+			return;
+		}
+
+		$asset_file   = Plugin::get_path( 'assets/js/admin-bar.asset.php' );
+		$asset        = require $asset_file;
+		$dependencies = $asset['dependencies'];
+		$version      = $asset['version'];
+
+		// Enqueue scripts.
+		wp_enqueue_script(
+			'amp-qa-tester-admin-bar-script',
+			Plugin::get_asset_url( 'js/admin-bar.js' ),
+			$dependencies,
```

Comment:
By adding the `admin-bar` as a dependency, this will be ensured to be included in AMP dev mode:

```suggestion
			array_merge( [ 'admin-bar' ], $dependencies ],
```